packages/jaeger-ui/src/model/link-patterns.js

```diff
@@ -0,0 +1,215 @@
+// @flow
+
+// Copyright (c) 2017 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import _uniq from 'lodash/uniq';
+import { getConfigValue } from '../utils/config/get-config';
+import type { Span, Trace } from '../types';
+
+const parameterRegExp = /\$\{([^{}]*)\}/g;
+
+type ProcessedTemplate = {
+  parameters: string[],
+  template: (data: { [parameter: string]: any }) => string,
```

Comment:
I think this can be 

```
template: { [string]: any } => string,
```

Reply:
Also, would `mixed` fit the bill instead of `any`?

https://flow.org/en/docs/types/mixed/

Reply:
Ok